### PR TITLE
#175636829 Configure BareFoot Nomad to use PostgreSQL as DB Engine

### PR DESCRIPTION
**Pull request by Thiernope**
#### What does this PR do?
Connect BareFoot Nomad to use PostgreSQL as DB Engine

#### Description of Task to be completed?
. run `npm install pg` to install postgreSQL to run in local machine  
. run `npm install --save pg pg-hstore` to install sequelize package  
. connect tech-strikers-bn-backend to postgreSQL dabase engine using sequelize  
. use the function `sequelize.authentication() `to test the connection

#### How should this be manually tested?
. run `git clone https://github.com/atlp-rwanda/tech-strikers-bn-backend.git`  
. run `cd tech-strikers-bn-backend`
. run `npm run dev`

#### Any background context you want to provide?
Hound is still complaining on the issue that is in the file which is already deleted.

#### What are the relevant pivotal tracker stories?
[#175636829 ](https://www.pivotaltracker.com/n/projects/2475704/stories/175636829)
#### Screenshots (if appropriate)
Connection message

### CODE REVIEW

File: src/database/config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { Pool, Client } = require("pg");
+
+const pool = new Pool({
```
[the remark this answers is not in the document]
Comment (MChristian00) [reply src/database/config.js:3]: Add a newline at the end of the file.

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const presets = [
+    [
+      "@babel/env",
+    ],
+  ];
+  
+  module.exports = { presets };
```
[the remark this answers is not in the document]
Comment (MChristian00) [reply babel.config.js:7]: Add a new line at the end of this file.

**Review by MChristian00 — CHANGES_REQUESTED** [review]
Great work, may you have the following resolved:
- Do a rebase to remove conflicts
- Check the files and add a newline at the end of the file where necessary.
- Ensure you have one commit.

If you have trouble resolving these, I'm just a message/call away to help you resolve them ✊🏽.

File: babel.config.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+
```
Comment (Duncanwar) [inline babel.config.js]: add a new line here

File: src/database/config.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  user: "andela",
+  host: "localhost",
+  database: "barefootnomad_db",
+  password: "andela2020",
+  port: "5432"
+});
+
+
+module.exports = pool;
```
Comment (Duncanwar) [inline src/database/config.js]: add a new line

**Review by Duncanwar — COMMENTED** [review]
Rebase your commit into one commit

**Comment by Duncanwar** [comment]
Resolve the conflicts locally

File: babel.config.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+const presets = [
+    [
+      "@babel/env",
+    ],
+  ];
+  
+  module.exports = { presets };
```
Comment (Duncanwar) [inline babel.config.js]: add new line

File: src/database/config.js (hunk 4 above)
Comment (Duncanwar) [reply src/database/config.js]: add a new line on line 12

**Review by Duncanwar — COMMENTED** [review]
Implement the feedback, I have given

File: .eslintrc.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+};
+
+
```
Comment (Duncanwar) [inline .eslintrc.js]: remove one line

Comment (Thiernope) [reply .eslintrc.js]: Hello Duncan, I am done to add only one line at the end. Thanks

File: package.json [hunk 7]
```diff
@@ -20,22 +20,23 @@
...
+    "express": "^4.17.1",
+    "express-jwt": "^5.3.3",
+    "express-session": "^1.17.1",
+    "jsonwebtoken": "^8.5.1",
     "method-override": "^2.3.10",
     "methods": "^1.1.2",
-    "mongoose": "^5.2.2",
-    "mongoose-unique-validator": "^2.0.1",
+    "mongoose": "^5.10.13",
+    "mongoose-unique-validator": "^2.0.3",
```
Comment (kipropbrian) [inline package.json]: Remove Mongoose, we do not need it

File: .eslintrc.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+    "extends": [
+        "airbnb-base"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+};
```
Comment (kipropbrian) [inline .eslintrc.js]: We already have a .eslintrc.json

**Review by kipropbrian — APPROVED** [review]
Much better

**Review by Duncanwar — APPROVED** [review]

**Pull request merged**